managed/services/telemetry/config.default.yml

```diff
@@ -2,7 +2,7 @@ telemetry:
   # PMM - generic information
   - id: PMMServerIAEnabled
     source: PMMDB_SELECT
-    query: (CASE WHEN alerting->'disabled' = 'false' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'alerting') AS ia
+    query: (CASE WHEN ia -> 'enabled' = 'true' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'ia') AS ia
```

Comment:
`ia` in settings was replaced with `alerting`. This change was added in 2.32 as I remember.

Reply:
for some reason we have old grafana in dev-container (grafana 8.3.10), while it should be 9.2.5.